.circleci/config.yml

```
@@ -1890,8 +1881,14 @@ workflows:
       # build-only
       - b_docs: *requires_nothing
       - b_ubu_ossfuzz: *requires_nothing
-      - b_ubu_2204: *requires_nothing
-      - b_ubu_2204_clang: *requires_nothing
+      - b_ubu_min_req_clang: *requires_nothing
+
+      # build and test with minimum supported versions of dependencies
+      - b_ubu_min_req: *requires_nothing
```

I think that it makes more sense to group `min_req` jobs together than to put `b_ubu_min_req_clang` under `# build-only`.

```suggestion
      # build and test with minimum supported versions of dependencies
      - b_ubu_min_req_clang: *requires_nothing
      - b_ubu_min_req: *requires_nothing
```

That was part of my suggestion in https://github.com/argotorg/solidity/pull/16177#discussion_r2364166235, but maybe it wasn't clear because it was mixed in with the 22.04 removal.

---

ahhh yeah, completely missed that!